pom.xml

```diff
@@ -2325,7 +2325,7 @@
               <groupId>org.apache.maven.plugins</groupId>
               <artifactId>maven-surefire-plugin</artifactId>
               <configuration>
-                <excludedGroups>external, large-disk-resource</excludedGroups>
+                <excludedGroups>external, large-disk-resource, stress, slow</excludedGroups>
```

Comment:
Does nightly use the `ci` profile?
If so, this is not the right place for this.